ports/qt5-base/portfile.cmake

```
@@ -45,6 +45,10 @@ else()
     )
 endif()
 
+if("postgresqlplugin" IN_LIST FEATURES)
```

```suggestion
set(WITH_PGSQL)
if("postgresqlplugin" IN_LIST FEATURES)
```

need to make sure nobody is messing with it in the triplet

---

Ok. This could be fixed for all other variables also.

---

yeah looks like `QT_OPENSSL_LINK` and `QT_MYSQL_PLUGIN` should use set and not `option`. For consistency you probably want to rename this variable to `QT_PGSQL_PLUGIN `

---

Agreed that these additional changes would be good, but don't seem critical enough to block merging this PR.